sigs.yaml

```diff
@@ -1,4 +1,26 @@
 sigs:
+- dir: sig-community
+  name: Community
+  mission_statement: >
+    The SIG's mission is to care about all aspects of the user and contributor community around the Operate First (Op1st) project.
+  charter_link: charter.md
+  label: community
+  leadership:
+    chairs:
+    - github: tbd  # TODO: fill in
```

Comment:
I propose @quaid as chair.
Any other volunteers?

Reply:
I would like to nominate @hpdempsey, @sallyom, @billburnseh , would any of you like to lead this? I think you would be a great fit!

Reply:
Inaugural meeting consensus is to have @schwesig & @quaid for initiating chairs

Reply:
```suggestion
    - github: tbd  # TODO: @quaid @schwesig 
```